RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -223,14 +223,22 @@ boolean LX_getStarKey()
 	
 	LX_buyStarKeyParts();
 
+	// summon Skinflute or Camel's Toe to get both stars and lines. We can copy them into delay zones like the 8-bit realm.
+	int copiesNeeded = max((8 - item_amount($item[star])) / 2, (7 - item_amount($item[line])) / 2);
```

Comment:
Could this be moved to `LX_summonMonster()`? Nice to have all in one spot, but if there is a functional/performance issue with moving it I'm game to keep it here

Reply:
Honestly, `LX_SummonMonster()` shouldn't exist. Things to advance quests should go in the relevant quest functions because otherwise, in a codebase of this size it becomes an onerous task for anyone working on the codebase to remember where everything is and what everything does.
Also we will end up duplicating code just to make sure we're not doing things when we don't mean to. And then we will end up with side-effects from resources being used incorrectly because the game changed something or a new IotM added something we consider in one place but whoever updated that place didn't change all the other places it needs changed.

Reply:
Yes this is a good point. Feel like there is lots of duplicating as well, so totally agree with ya